low-level/sspl_ll_d

```diff
@@ -713,6 +715,14 @@ def send_thread_controller_request(state):
 #     logger.info(f"Received SIGUSR1 signal to set log level to '{log_level}'")
 #     logger.setLevel(log_level)
 
+def initialize_iem():
+    try:
+        EventMessage.init(component='sspl', source='S')
+        logger.info("IEM framework initiated successfully!!!")
+    except EventMessageError as e:
+        logger.error("Failed to initialize IEM Framework."
```

Comment:
so if Init fails for some reason, SSPL still later tries to generate IEM irrespective of whether init had failed or not. We need to relate this failure ideally, may be through a marker file with sspl pid in it. and retry init just before sending IEM ..

Reply:
Agree @cdeshmukh.
so in case of iem init failure, it will create a file at loc:-

> IEM_INIT_FILE = "/var/%s/sspl/data/iem/iem_init" %(PRODUCT_FAMILY)

and iem before sending the alert, will check for this file, if it is present, it will retry to initialize the IEM framework & then will send the alert and remove this file.